corehq/apps/app_manager/templates/app_manager/releases.html

```diff
@@ -318,8 +322,8 @@ <h6 data-bind="if: !built_with.signed()">{% trans "Unsigned Jar" %}</h6>
                     <div class="accordion-group">
                         <div class="accordion-heading">
                             <a class="accordion-toggle" data-bind="
+                                {# TODO: This tracking code is probably rarely fired anymore because now this accordion is open by default #}
```

Comment:
fine to remove from my perspective

On Wed, Apr 29, 2015 at 10:34 AM, Cory Zue <notifications@github.com> wrote:

> In corehq/apps/app_manager/templates/app_manager/releases.html
> <https://github.com/dimagi/commcare-hq/pull/6492#discussion_r29341283>:
>
> > @@ -318,8 +322,8 @@ <h6 data-bind="if: !built_with.signed()">{% trans "Unsigned Jar" %}</h6>
> >                      <div class="accordion-group">
> >                          <div class="accordion-heading">
> >                              <a class="accordion-toggle" data-bind="
> > +                                {# TODO: This tracking code is probably rarely fired anymore because now this accordion is open by default #}
>
> @amsagoff <https://github.com/amsagoff> should we just remove this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/6492/files#r29341283>.
>